doc/source/operations/tempest.rst

```diff
@@ -0,0 +1,303 @@
+    $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+    sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+Installing Docker on CentOS/Rocky:
+
+.. code-block:: bash
+
+    sudo yum install -y yum-utils
+    sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+
+Ensure Docker is running:
+
+.. code-block:: bash
+
+    sudo systemctl start docker
+
+The Docker ``buildx`` plugin must be installed. If you are using an existing
+installation of docker, you may need to install it with:
+
+.. code-block:: bash
+
+    sudo dnf/apt install docker-buildx-plugin
+    sudo docker buildx install
+    # or if that fails:
+    sudo docker plugin install buildx
+
+Build a Kayobe automation image:
+
+.. code-block:: bash
+
+    git submodule init
+    git submodule update
+    # If running on Ubuntu, the fact cache can confuse Kayobe in the CentOS-based container
+    mv etc/kayobe/facts{,-old}
+    sudo DOCKER_BUILDKIT=1 docker build --file .automation/docker/kayobe/Dockerfile --tag kayobe:latest .
+
+Configuration
+=============
+
+Kayobe automation configuration files are stored in the ``.automation.conf/``
+directory. It contains:
+
+- A script used to export environment variables for meta configuration of
+  Tempest - ``.automation.conf/config.sh``.
+- Tempest configuration override files, stored in ``.automation.conf/tempest/``
+  and conventionally named ``tempest.overrides.conf`` or
+  ``tempest-<environment>.overrides.conf``.
+- Tempest load lists, stored in ``.automation.conf/tempest/load-lists``.
+- Tempest skip lists, stored in ``.automation.conf/tempest/skip-lists``.
+
+config.sh
+---------
+
+``config.sh`` is a mandatory shell script, primarily used to export environment
+variables for the meta configuration of Tempest.
+
+See:
+https://github.com/stackhpc/docker-rally/blob/master/bin/rally-verify-wrapper.sh
+for a full list of tempest parameters that can be overridden.
+
+The most common variables to override are:
+
+- ``TEMPEST_CONCURRENCY`` - The maximum number of tests to run in parallel at
+  one time. Higher values are faster but increase the risk of timeouts. 1-2 is
+  safest in CI/Tenks/Multinode/AIO etc. 8-32 is typical in production. Default
+  value is 2.
+- ``KAYOBE_AUTOMATION_TEMPEST_LOADLIST``: the filename of a load list in the
+  ``load-lists`` directory. Default value is ``default`` (symlink to refstack).
+- ``KAYOBE_AUTOMATION_TEMPEST_SKIPLIST``: the filename of a load list in the
+  ``skip-lists`` directory. Default value is unset.
+- ``TEMPEST_OPENRC``: The **contents** of an ``openrc.sh`` file, to be used by
+  Tempest to create resources on the cloud. Default is to read in the contents
+  of ``etc/kolla/public-openrc.sh``.
+
+tempest.overrides.conf
+----------------------
+
```

Comment:
Should this section mention [Shakespeare](https://github.com/stackhpc/shakespeare)?

Reply:
I did look into it but to me it actually seemed way more complex than just setting the config manually. I don't think I'm the right person to add that section. If we want to include it, should probably be a follow-up PR by someone who's got some more experience there.

Reply:
Perhaps you could just link to it as a source of best practice config?

Reply:
Done